### PR TITLE
Add OpenShift template

### PR DESCRIPTION
**Pull request by Chathuru**
WSO2 Microgateway OpenShift install.

### CODE REVIEW

File: Template.yaml [hunk 1]
```diff
@@ -0,0 +1,648 @@
+apiVersion: v1
```
Comment (ThilinaManamgoda) [inline Template.yaml]: Shall we add the license header in this file

File: Template.yaml [hunk 2]
```diff
@@ -0,0 +1,648 @@
...
+#    password: ${{LOGSTASH_PASSWORD}}
+#    username: ${{LOSTASH_USERNAME}}
+
+# - apiVersion: v1
+#   kind: ConfigMap
+#   metadata:
+#     name: logstash-conf
+#   data:
+#     logstash.conf: |
+#       # Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (ThilinaManamgoda) [inline Template.yaml]: Let's remove this license header from this since it's should only be defined once for the file.

File: Template.yaml [hunk 3]
```diff
@@ -0,0 +1,648 @@
...
+#           }
+#       }
+
+# - apiVersion: v1
+#   kind: ConfigMap
+#   metadata:
+#     name: logstash-yml
+#   data:
+#     logstash.yml: |
+#       # Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (ThilinaManamgoda) [inline Template.yaml]: Remove license header

File: Template.yaml (hunk 1 above)
Comment (Chathuru) [reply Template.yaml]: Resolve with the commit 70554b4

File: Template.yaml [hunk 4]
```diff
@@ -0,0 +1,634 @@
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (chirangaalwis) [inline Template.yaml]: @Chathuru please update the license header year to latest.

**Comment by Chathuru** [comment]
Issue https://github.com/wso2/openshift-microgateway/issues/2 resolved

**Pull request merged**